lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -633,6 +633,19 @@ radius_server:
     config_set: '<state> radius-server host <ip> key <key>'
     default_value: null
 
+radius_server_group:
```

Comment:
radius_server_group has several commands in the sub-menu besides 'hosts', so using a _template definition makes sense and should simplify the code; e.g.:

```
radius_server_group:
  _template:
    config_get: "show running-config radius all"
    config_get_token: '/^aaa group server radius (\S+)/'
    config_set: '<state> aaa group server radius <name>'

  hosts:
    config_get_token_append: '/server ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/']
    config_set_append: '<state> server <ip>'
    default_value: ''

  foo:
    config_get_token_append: '/foo (.*)/'
    config_set_append: '<state> foo <bar>'
    default_value: ''

```